assets/component-mega-menu.css

```diff
@@ -49,13 +42,22 @@
 .mega-menu__link {
   color: rgba(var(--color-foreground), 0.75);
   display: block;
+  font-size: 1.2rem;
   line-height: calc(1 + 0.3 / var(--font-body-scale));
-  padding-bottom: 0.8rem;
-  padding-top: 0.8rem;
+  padding-bottom: 0.6rem;
+  padding-top: 0.6rem;
   text-decoration: none;
   transition: text-decoration var(--duration-short) ease;
 }
 
+.mega-menu__link--level-2 {
+  font-size: 1.4rem;
+}
+
+.mega-menu__link--level-2:not(:only-child) {
+  margin-bottom: 1rem;
```

Comment:
Looks better now 👌 

But I think maybe it could be a little less margin.
![](https://screenshot.click/28-47-9ki39-v5i59.png)

Reply:
I also think they can be the same value!